xen/drivers/cpufreq/cpufreq_vscmi.c

```diff
@@ -0,0 +1,115 @@
+/*
+ *  xen/drivers/cpufreq/cpufreq_vscmi.c
+ *
+ *  Copyright (C)  2019 Volodymyr Babchuk <volodymyr_babchuk@epam.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
```

Comment:
I am afraid, something is missed here. Please follow the document http://xenbits.xen.org/gitweb/?p=xen.git;a=blob;f=CONTRIBUTING

Reply:
Oh, okay